zerver/openapi/zulip.yaml

```diff
@@ -7910,71 +7915,12 @@ paths:
-                                  opposite value, `in_home_view=!is_muted`); for
-                                  backwards-compatibility, modern Zulip still accepts/returns that property.
-                                - `pin_to_top`: Whether to pin the stream at the top of the stream list.
-                                - `desktop_notifications`: Whether to show desktop notifications for all
-                                  messages sent to the stream.
-                                - `audible_notifications`: Whether to play a sound notification for all
-                                  messages sent to the stream.
-                                - `push_notifications`: Whether to trigger a mobile push notification for
-                                  all messages sent to the stream.
-                                - `email_notifications`: Whether to trigger an email notification for all
-                                  messages sent to the stream.
-                                - `wildcard_mentions_notify`: whether wildcard mentions trigger notifications
-                                  as though they were personal mentions in this stream.
-                              enum:
-                                - color
-                                - is_muted
-                                - pin_to_top
-                                - desktop_notifications
-                                - audible_notifications
-                                - push_notifications
-                                - email_notifications
-                                - wildcard_mentions_notify
-                                - in_home_view
-                            value:
-                              description: |
-                                The desired value of the property.
-                              oneOf:
-                                - type: boolean
-                                - type: string
-                        description: |
-                          The same `subscription_data` array sent by the client for the request.
+                      ignored_parameters_unsupported:
+                        $ref: "#/components/schemas/IgnoredParametersUnsupported"
                     example:
                       {
-                        "subscription_data":
-                          [
-                            {
-                              "stream_id": 1,
-                              "property": "pin_to_top",
-                              "value": true,
-                            },
-                            {
-                              "stream_id": 3,
-                              "property": "color",
-                              "value": "#f00f00",
-                            },
-                          ],
+                        "ignored_parameters_unsupported":
+                          ["data_subscriptions"],
```

Comment:
I'd call this `invalid_parameter`, just to make clear that is its purpose here.

Reply:
Done. Also made the same change in for the value in the test as well.